### PR TITLE
/volunteer

### PR DESCRIPTION
**Pull request by JFernandez1995**
/volunteer page added

### CODE REVIEW

File: HackMerced/2017F/src/app/src/components/forms/volunteer.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ *
+ */
+
+import React, { Component } from 'react';
+import { TextInputBlock } from '../partials';
+import { updateVolForm} from '../../actions';
+
+const assign = Object.assign || require('object.assign');
+
+export class VolForm extends Component {
```
Comment (4shub) [inline HackMerced/2017F/src/app/src/components/forms/volunteer.js]: Can you use full names (VolunteerForm)

File: HackMerced/2017F/src/app/src/components/volunteer.js [hunk 2]
```diff
@@ -1,13 +1,31 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router'
-import { Header, PatternScaffold, LogoWithCopy } from './partials'
+import React, { Component} from 'react';
+import { connect } from 'react-redux';
+import { browserHistory } from 'react-router'
+import { VolForm } from './forms';
```
Comment (4shub) [inline HackMerced/2017F/src/app/src/components/volunteer.js]: Same

File: HackMerced/2017F/src/app/src/reducers/index.js [hunk 3]
```diff
@@ -32,6 +32,16 @@ const initialState = {
   },
 
 
+
+volunteerPersonForm: {
+  name: '',
+  age: '',
+  avail: '',
```
Comment (4shub) [inline HackMerced/2017F/src/app/src/reducers/index.js]: use full names please

File: HackMerced/2017F/src/app/src/reducers/index.js [hunk 4]
```diff
@@ -32,6 +32,16 @@ const initialState = {
   },
 
 
+
+volunteerPersonForm: {
+  name: '',
+  age: '',
+  avail: '', 
+  diet: '',
```
Comment (4shub) [inline HackMerced/2017F/src/app/src/reducers/index.js]: call this dietary_restrictions

File: HackMerced/2017F/src/app/src/reducers/index.js [hunk 5]
```diff
@@ -32,6 +32,16 @@ const initialState = {
...
+  size: '',
```
Comment (4shub) [inline HackMerced/2017F/src/app/src/reducers/index.js]: shirt_size

**Review by 4shub — CHANGES_REQUESTED** [review]
Pls fix

**Comment by JFernandez1995** [comment]
fixed naming conventions for /volunteer

**Comment by zzmarkzz321** [comment]
@JFernandez1995 Please fix the merge conflicts

**Review by 4shub — APPROVED** [review]

**Pull request merged**